src/backends/github/API.js

```diff
@@ -85,8 +86,14 @@ export default class API {
     });
   }
 
+  truncateSlugForBranchName(slug) {
+    const slugMaxLength = this.branchNameMaxLength - CMS_BRANCH_PREFIX.length;
+    return slug.substring(0, slugMaxLength);
```

Comment:
👍 Nitpick though, I thought `substr` is proper here instead of `substring` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring).